### ISSUE TITLE
Spanische Titel

### ISSUE THREAD

**Issue by K3vb3rt** [PCJones/radarr-sonarr-german-dual-language#23]
Erstmal besten Dank für die klasse Anleitung!
Aber irgendwie bekomme ich es nicht hin, dass er NUR deutsche Titel lädt - ich möchte keine Backup-Tonspur, ausschließlich deutsche Titel.

In diesem Fall lädt er mir einen spanischen Titel herunter :-D

<img width="1242" alt="Bildschirmfoto 2024-08-18 um 10 10 11" src="https://github.com/user-attachments/assets/27d746f5-6e03-4344-bb45-d64060da4b8f">

Hier mein Qualitätsprofil - hoffentlich alles richtig drin:

<img width="451" alt="Bildschirmfoto 2024-08-18 um 10 10 28" src="https://github.com/user-attachments/assets/e2a81025-37ab-4a3b-963f-902d1f0f0f17">

**Comment by K3vb3rt** [comment]
@PCJones Hast du vielleicht eine Idee was ich hier machen kann?

**Comment by PCJones** [comment]
Sorry für die späte Antwort. Du hast nicht wie im guide angegeben alle Qualitäten in eine Gruppe zusammengefügt, ggf. liegt es daran?

**Comment by PCJones** [comment]
Das hier könnte es auch sein:

1.) Go to Settings->Media Management
2.) Make sure Advanced Settings are enabled at the top
3.) Scroll down to File Management
4.) Set Propers and Repacks to Do not Prefer

**Comment by PCJones** [comment]
> Und du wirst bei "nur deutsche Tonspur" gerade für aktuelle Filme wahrscheinlich öfter Schwierigkeiten haben, weil viele Releases Deutsch und Englisch haben.

Er sucht ja aber auch nach "German DL" bzw bevorzugt das sogar. Das ist also kein Problem

**Comment by K3vb3rt** [comment]
Das war jetzt auch der einzige Fall wo das vorkam. Die Gruppierung hatte ich aber vorgenommen.Zwar etwas modifiziert, aber grundsätzlich schon wie im Guide:

<img width="427" alt="Bildschirmfoto 2024-09-04 um 17 01 39" src="https://github.com/user-attachments/assets/406bd912-80ee-45f4-aa09-f6a3f6fc95f4">

Hier noch ein Screenshot der interaktiven Suche:

![Uploading Bildschirmfoto 2024-09-04 um 17.02.33.png…]()

**Comment by PCJones** [comment]
> Das war jetzt auch der einzige Fall wo das vorkam. Die Gruppierung hatte ich aber vorgenommen.Zwar etwas modifiziert, aber grundsätzlich schon wie im Guide:
> <img alt="Bildschirmfoto 2024-09-04 um 17 01 39" width="427" src="https://private-user-images.githubusercontent.com/60546306/364433476-406bd912-80ee-45f4-aa09-f6a3f6fc95f4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU2NDk5OTMsIm5iZiI6MTcyNTY0OTY5MywicGF0aCI6Ii82MDU0NjMwNi8zNjQ0MzM0NzYtNDA2YmQ5MTItODBlZS00NWY0LWFhMDktZjZhM2Y2ZmM5NWY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA2VDE5MDgxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhZDhkYmE1NDU0MjZkZmZiNDI3NTgwZDE1MjFlOGZhYmRkMzI1ZDAzMWI2MzM4MDc4Mzk2NmRiYWE1ZTEyZTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QegnhKt0E6YsweKxUXA59_o5yTlBxkyRORqjIL4pM7s">
> 
> Hier noch ein Screenshot der interaktiven Suche:
> <img alt="Bildschirmfoto 2024-09-04 um 17 02 33" width="1438" src="https://private-user-images.githubusercontent.com/60546306/364433999-1c747193-db2d-4185-b3b7-b83382914885.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjU2NDk5OTMsIm5iZiI6MTcyNTY0OTY5MywicGF0aCI6Ii82MDU0NjMwNi8zNjQ0MzM5OTktMWM3NDcxOTMtZGIyZC00MTg1LWIzYjctYjgzMzgyOTE0ODg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTA2VDE5MDgxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA3Y2JhMTgyOWIzYjcxNzVhYzdlYzc2NjVmMGRiMDIzNzk4NDhkYzM2ZDBhY2E5MDEzMzBhZjk0ZjM5ZWZhNzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.EjhbkqPdFFnCrZlxU2B8Ci6wIMXKR-PxhDS9GVSkHAo">

Es ist eben nicht wie im Guide, da du 3 verschiedene aktive Gruppen hast. Es müssen jedoch alle Qualitäten die du haben möchtest in einer Gruppe sein. Sonst kann es eben sein, dass es z.B. Bluray-1080p in German(DL) gibt, jedoch ein spanisches Release gezogen wird, da es in einer höheren Qualität (z.B. WEBDL-2160p) vorhanden ist.

**Comment by K3vb3rt** [comment]
Dann hab ich den Guide wohl falsch verstanden. Ist das dann jetzt hier so richtig? Dürfte ja nur noch eine aktive Gruppe darstellen. ![image](https://github.com/user-attachments/assets/2a5b169c-b588-4f48-9648-f06472e186b0)

**Comment by PCJones** [comment]
Ja, fast. Innerhalb der Gruppe sollte die Reihenfolge trotzdem stimmen, also WEBDL-1080p vor HDTV und WebRip. Sonst passt es aber
